src/main/java/htsjdk/samtools/SQTagUtil.java

```diff
@@ -27,7 +27,9 @@
  * Utility methods for encoding and decoding the SQ tag value of SAMRecord.
  * 
  * @author alecw@broadinstitute.org
+ * @deprecated since 11/2018. SQ is a reserved tag that shouldn't be used and this code untested.
```

Comment:
```suggestion
 * @deprecated since 11/2018. SQ is a deprecated tag that should no longer be used.  This code is untested and and unsupported.  There is no replacement for these utilities.
```